### PR TITLE
fix: scale mousepoint(#550)

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

#550 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/src/charts/chart.ts [hunk 1]
```diff
@@ -279,11 +279,15 @@ export default abstract class Chart<T extends Options> {
 
     const delegationMethod = `on${eventType[0].toUpperCase() + eventType.substring(1)}`;
 
-    const canvasRect = this.painter.ctx.canvas.getBoundingClientRect();
+    const canvas = this.painter.ctx.canvas;
+    const canvasRect = canvas.getBoundingClientRect();
```
Comment (lja1018) [inline apps/chart/src/charts/chart.ts:283]: 바로 `{width, height, ...}` 꺼내써도 될것 같습니다!

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
확인했습니다~!

File: apps/chart/src/charts/chart.ts (hunk 1 above)
Comment (jung-han) [reply apps/chart/src/charts/chart.ts:283]: 하단에 사용하고 있는 부분이 있어서 여기서는 뎁스가 깊지 않아 디스트럭처링을 굳이 하지 않아도 될 것 같아요~

**Pull request merged**